src/ProjectTemplates/test/WorkerTemplateTest.cs

```diff
@@ -21,13 +21,18 @@ public WorkerTemplateTest(ProjectFactoryFixture projectFactory, ITestOutputHelpe
         public ProjectFactoryFixture ProjectFactory { get; }
         public ITestOutputHelper Output { get; }
 
-        [ConditionalFact]
+        [ConditionalTheory]
         [OSSkipCondition(OperatingSystems.Linux, SkipReason = "https://github.com/dotnet/sdk/issues/12831")]
-        public async Task WorkerTemplateAsync()
+        [InlineData("C#")]
+        [InlineData("F#")]
+        [Repeat]
+        public async Task WorkerTemplateAsync(string language)
         {
-            Project = await ProjectFactory.GetOrCreateProject("worker", Output);
+            Project = await ProjectFactory.GetOrCreateProject(
+                $"worker-{ language.ToLowerInvariant()[0] }sharp",
+                Output);
 
-            var createResult = await Project.RunDotNetNewAsync("worker");
+            var createResult = await Project.RunDotNetNewAsync("worker", language: language);
```

Comment:
I recall the F# MVC templates do not completely run because of issues we ran into with the F# compiler. Nice that it works, but just something to be aware of

Reply:
Which issues? Happy to investigate